python/pyspark/ml/torch/tests/test_distributor.py

```diff
@@ -288,6 +288,13 @@ def test_local_training_succeeds(self) -> None:
                 if cuda_env_var:
                     self.delete_env_vars({CUDA_VISIBLE_DEVICES: cuda_env_var})
 
+    def test_local_file_with_pytorch(self) -> None:
```

Comment:
As a sidenote, used `local-cluster` for testing `test_local_file_with_pytorch` and `test_distributed_file_with_pytorch`.

Reply:
We should skip these tests when `pytorch` is not installed, @rithwik-db .